app/api/__init__.py

```diff
@@ -34,7 +35,7 @@ def check_csrf_token():
 
         if not (new_csrf_valid):
             rollbar.report_message('csrf.invalid_token: Aborting request check_csrf_token()', 'error', request)
-            abort(400, 'Invalid CSRF token. Please try again.')
+            abort('Invalid CSRF token. Please try again.')
```

Comment:
By using our `abort()` wrapper it will log the message of a CSRF error, so we can keep track of them easily in our dashboards.